iquip/apps/explorer.py

```diff
@@ -56,9 +94,12 @@ def loadFileTree(self):
         It assumes that all experiment files are in self.repositoryPath.
         """
         self.explorerFrame.fileTree.clear()
-        threading.Thread(
-            target=lambda: self._addFile(self.repositoryPath, self.explorerFrame.fileTree)
-        ).start()
+        self.thread = FileFinderThread(
+            self.repositoryPath,
+            self.explorerFrame.fileTree,
+            self._addFile
+        )
```

Comment:
Please add the `parent=self` argument for a `QObject`.
See also: https://doc.qt.io/qt-5/qthread.html#details

Reply:
Oh, I was completely missing it. I will add the argument.